keyboards/stront/keymaps/hid/config.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 zzeneg (@zzeneg)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* HID is supported only on master, so left must be master */
```

Comment:
Why?  

You could use the custom data sync code to sync the data to the other half, if need be.  I do this with a number of things, such as the oled keylogger char array. 

for reference: 
https://docs.qmk.fm/#/feature_split_keyboard?id=custom-data-sync

https://github.com/qmk/qmk_firmware/blob/16767e4d59c2334fcd2d5e6556a68d5ff60ffd7b/users/drashna/split/transport_sync.c
Specifically, check the sections wrapped by `CUSTOM_OLED_DRIVER` for reference.

Reply:
Thanks! Honestly I wanted to avoid it as I thought it would be much more complicated. Turned out quite easy though and also keyboard is more performant if the half with touchpad is used as a master.

Reply:
Yeah, it's not as complex at it seems and works well.

As for the trackpad, on splits, it's throttled to 10ms between tasks.  You can fine tune this, and set `POINTING_DEVICE_TASK_THROTTLE_MS` to 1 (to 1ms), and that may boost performance.

Reply:
According to docs (and code) it's already set to 1ms if `SPLIT_POINTING_ENABLE` is defined. So we're good there.
I was mostly talking about scan rate, which goes from 1100ms with display half as master to 1400ms with trackpad one :)